### ISSUE TITLE
[BUG] unquoted quote ' in variable  (Postgresql)

### ISSUE THREAD

**Issue by eMerzh** [grafana/grafana#12785]
Hi,

I tried to use a variable wich contains names
and it works well, but if one of the name have a single quote in it, it breaks my query

query : 
```sql
select count(*) 
from client
where c.name in($client)
 ```

list of values in $client is ~:
- first
- second
- it's the third

the query produced is
```sql
select count(*) 
from client
where c.name in ('it's the third')
 ```
which is not right.

Grafana v5.2.0 (ad4d717) using postgreql

**Comment by ronansalmon** [comment]
Hi, this is also happening with datasource SQL Server : https://community.grafana.com/t/sql-server-escape-queries/9152

**Comment by eMerzh** [comment]
yeah the issue (about pg at least) lies arround https://github.com/grafana/grafana/blob/27835f373e581430aab7763eb08075fb308c8353/public/app/plugins/datasource/postgres/datasource.ts#L19 and a few line below...

**Comment by marefr** [comment]
Fixed by #12791
